code/modules/clothing/suits/jobs.dm

```diff
@@ -151,6 +157,21 @@
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO
 	allowed = list(/obj/item/kitchen/knife)
 
+//Chief Engineer
+/obj/item/clothing/suit/mantle/chief_engineer
+	name = "chief engineer's mantle"
+	desc = "A slick, authoritative cloak designed for the Chief Engineer."
+	icon_state = "cemantle"
+	item_state = "cemantle"
+	allowed = list(/obj/item/flashlight,/obj/item/tank,/obj/item/t_scanner, /obj/item/rcd)

```

Comment:
I don't mean to be a stickler but could we get spacing after the commas? Style guide blah blah blah